charts/open-notificaties/values.yaml

```diff
@@ -89,13 +93,46 @@ affinity: {}
 
 worker:
   replicaCount: 1
+  podLabels: {}
+  existingSecret: null
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+
+flower:
+  enabled: true
+  replicaCount: 1
+  podLabels: {}
+  existingSecret: null
+  service:
+    type: ClusterIP
+    port: 80
+  livenessProbe:
+    initialDelaySeconds: 60
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+  readinessProbe:
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
   resources: {}
   autoscaling:
     enabled: false
     minReplicas: 1
     maxReplicas: 100
     targetCPUUtilizationPercentage: 80
     # targetMemoryUtilizationPercentage: 80
+  envVars: 
+    FLOWER_URL_PREFIX: flower
```

Comment:
Can we make this configuration option explicit in `.Values.flower.settings.urlPrefix`? Think we can rename `envVars` to `extraEnvVars` then.

Reply:
You mean making _just this one_ explicit right? As there are many other Flower config parameters possible, I'd rather not make them all explicit as this gives us a maintenance burden ;-) 

Furthermore, will rename `envVars` to `extraEnvVars`

Reply:
Fixed in:

- 6fbb79141ab9f97468d6759d51912181048724d7
- fa467ffac25077ccca23781a42aaa4869b246b07